pkg/errors/errors.go

```diff
@@ -653,6 +653,10 @@ var (
 		"changefeed update error: %s",
 		errors.RFCCodeText("CDC:ErrChangefeedUpdateRefused"),
 	)
+	ErrChangefeedUpdateFailedTransaction = errors.Normalize(
```

Comment:
error info should be added to the errors.toml file , run make check to fix the lint error

Reply:
fixed